### PR TITLE
Mention how to push to internal/admin branch

### PR DESCRIPTION
**Pull request by lwhsu**
This is useful for committer mentors.

### CODE REVIEW

File: URLs.md [hunk 1]
```diff
@@ -116,6 +116,16 @@ git checkout -b admin internal/admin
 For browsing `internal/admin` branch on web:
 https://cgit.freebsd.org/${repo}/log/?h=internal/admin
 
+For pushing, either specify the full refspec:
+```
+git push origin HEAD:refs/internal/admin
+```
+
+Or setting `push.default` gitconfig to use the `upstream`:
```
Comment (bsdimp) [inline URLs.md]: s/upstream/freebsd/ since that's what we're trying to use everywhere

File: URLs.md [hunk 2]
```diff
@@ -116,6 +116,16 @@ git checkout -b admin internal/admin
...
+```
+git config push.default upstream
```
Comment (bsdimp) [inline URLs.md:126]: mixed feelings about this. likely safe enough. will go along with group consensus.

File: URLs.md (hunk 1 above)
Comment (lwhsu) [reply URLs.md]: `upstream` here is the config value of `push.default`, to ask git push to where the branch came from:
https://git-scm.com/docs/git-config#Documentation/git-config.txt-pushdefault

Maybe change the wording like "setting `push.default` gitconfig to use `upstream` model"?

File: URLs.md (hunk 2 above)
Comment (lwhsu) [reply URLs.md:126]: I've tested in my end, pushed to the `main` and `internal/admin` branches, may need to test more with src workflow like vendor import and MFC.

**Comment by lwhsu** [comment]
What's the correct setting?

File: URLs.md (hunk 1 above)
Comment (delphij) [reply URLs.md]: Yes, `upstream` is the hardcoded value that git uses to represent the upstream branch from which the current working branch is created.

Comment (delphij) [reply URLs.md]: I'd probably amend this to something like:

Or setting `push.default` to `upstream` which will make `git push` to push the current branch back to its upstream by default, which is more suitable for our workflow:

**Comment by 0mp** [comment]
What's the status of this documentation?

It looks like there is a hook preventing me from pushing (most likely a good thing):

```console
$  git push freebsd HEAD:refs/internal/admin
Enumerating objects: 5, done.
Counting objects: 100% (5/5), done.
Delta compression using up to 8 threads
Compressing objects: 100% (3/3), done.
Writing objects: 100% (3/3), 421 bytes | 421.00 KiB/s, done.
Total 3 (delta 1), reused 0 (delta 0), pack-reused 0
remote:
remote: ================================================================
remote: ref = refs/internal/admin
remote: old = b4f794b7edd054e8fb96be7a6e8f846eda317538
remote: new = 42f51623f1091696c37907a4e8dad2f6a92a4b78
remote: repo = doc
remote: hash = 42f51623f1091696c37907a4e8dad2f6a92a4b78
remote: commit:
remote: tree 9dcea5f1c8ffa80f4a4e558f3f090187eb12c1cf
remote: parent b4f794b7edd054e8fb96be7a6e8f846eda317538
remote: author Mateusz Piotrowski <0mp@FreeBSD.org> 1609886883 +0100
remote: committer Mateusz Piotrowski <0mp@FreeBSD.org> 1609889084 +0100
remote:
remote: Release Daniel Ebdrup Jensen from mentorship
remote:
remote: Congratulations, Daniel! You are ready now to fly on your own.
remote:
remote: Approved by:    doceng (implicit)
remote:
remote: ================================================================
remote:
remote: FATAL: VREF/APPROVERS-CHECK/doc: helper program exit status 65280
remote: error: hook declined to update refs/internal/admin
To ssh://gitrepo.freebsd.org/doc.git
 ! [remote rejected]       HEAD -> refs/internal/admin (hook declined)
error: failed to push some refs to 'ssh://gitrepo.freebsd.org/doc.git'
```

**Comment by lwhsu** [comment]
> What's the status of this documentation?
> 
> It looks like there is a hook preventing me from pushing (most likely a good thing):
> 
> ```
> $  git push freebsd HEAD:refs/internal/admin
> Enumerating objects: 5, done.
> Counting objects: 100% (5/5), done.
> Delta compression using up to 8 threads
> Compressing objects: 100% (3/3), done.
> Writing objects: 100% (3/3), 421 bytes | 421.00 KiB/s, done.
> Total 3 (delta 1), reused 0 (delta 0), pack-reused 0
> remote:
> remote: ================================================================
> remote: ref = refs/internal/admin
> remote: old = b4f794b7edd054e8fb96be7a6e8f846eda317538
> remote: new = 42f51623f1091696c37907a4e8dad2f6a92a4b78
> remote: repo = doc
> remote: hash = 42f51623f1091696c37907a4e8dad2f6a92a4b78
> remote: commit:
> remote: tree 9dcea5f1c8ffa80f4a4e558f3f090187eb12c1cf
> remote: parent b4f794b7edd054e8fb96be7a6e8f846eda317538
> remote: author Mateusz Piotrowski <0mp@FreeBSD.org> 1609886883 +0100
> remote: committer Mateusz Piotrowski <0mp@FreeBSD.org> 1609889084 +0100
> remote:
> remote: Release Daniel Ebdrup Jensen from mentorship
> remote:
> remote: Congratulations, Daniel! You are ready now to fly on your own.
> remote:
> remote: Approved by:    doceng (implicit)
> remote:
> remote: ================================================================
> remote:
> remote: FATAL: VREF/APPROVERS-CHECK/doc: helper program exit status 65280
> remote: error: hook declined to update refs/internal/admin
> To ssh://gitrepo.freebsd.org/doc.git
>  ! [remote rejected]       HEAD -> refs/internal/admin (hook declined)
> error: failed to push some refs to 'ssh://gitrepo.freebsd.org/doc.git'
> ```

Looks like I haven't enabled this branch in approvers. I'll fix it soon.

**Comment by 0mp** [comment]
> Looks like I haven't enabled this branch in approvers. I'll fix it soon.

Cool! I'll try to push my change later on today. Thanks, @lwhsu!

**Pull request merged**